clang/include/clang/3C/ProgramInfo.h

```diff
@@ -151,6 +164,8 @@ class ProgramInfo : public ProgramVariableAdder {
   // Function to check if an external symbol is okay to leave constrained.
   bool isExternOkay(const std::string &Ext);
 
+
+
```

Comment:
Delete extra lines.

Reply:
Note that this can be done with `clang-format` if it bugs us enough to do it on every change.